RULES_DESCRIPTIONS.md

```diff
@@ -501,6 +502,23 @@ Example:
   arguments = ["This is the text that must appear at the top of source files."]
 ```
 
+## file-length-limit
+
+_Description_: This rule enforces a maximum number of lines per file, in order to aid in maintainability and reduce complexity.
+
+_Configuration_:
+
+* `max` (int) a maximum number of lines in a file (default 1000)
+* `skipComments` (bool) ignore lines containing just comments
+* `skipBlankLines` (bool) ignore lines made up purely of whitespace
```

Comment:
Please explain the default values for these two

Reply:
Extended slightly description